tests/std/tests/usual_17_matrix.lst

```diff
@@ -7,7 +7,7 @@ RUNALL_INCLUDE .\prefix.lst
 RUNALL_CROSSLIST
 PM_CL="/w14640 /Zc:threadSafeInit-"
 RUNALL_CROSSLIST

```

Comment:
This file begins with `# When updating this file, also update tests\P0088R3_variant\env.lst to match` (which I forgot about until reviewing #1162).

Reply:
I fixed the inconsistencies.